eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,19 +1,27 @@
 parameters:
   Directory: 'not-specified'
-  IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  
+  IgnoreLinksFile: '$(Build.SourcesDirectory)/eng/ignore-links.txt'
+  WorkingDirectory: '$(System.DefaultWorkingDirectory)'
+  ScriptDirectory: 'eng/common/scripts'
+  Recursive: $false
+  BranchReplaceRegex: '^$(System.PullRequest.SourceRepositoryURI).*/(?:blob|tree)/)master(/.*)$'
```

Comment:
Just to make sure this didn't get lost. We should use the env variable here at least for the CommitId so that this doesn't blow-up when we use this in non-PullRequests.

Reply:
Right, but env var does not work as I have ever tried. Need to experiment which is the right way to state it.